src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,3 +1,18 @@
 import React from 'react';
+import { User } from '../../types/user';
 
-export const UserInfo: React.FC = () => <>Put the user here</>;
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  if (user) {
+    return (
+      <a className="UserInfo" href={'mailto:' + user.email}>
+        {user.name}
+      </a>
+    );
+  }
+
+  return undefined;
```

Comment:
That's how i would do it. 
```suggestion
 return {user ? (
  <a className="UserInfo" href={'mailto:' + user.email}>
        {user.name}
  </a>
  ) : null}
```